doc/plugins/eu.esdihumboldt.hale.doc.user/html/resources/resources_HALE.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Resources and Support</title>
+</head>
+<body>
+	<h1>Resources and Support</h1>
+
+	<p>This section contains a list of links to a wide variety of helpful resources.</p>
+
+	<a href="PLUGINS_ROOT/eu.esdihumboldt.hale.doc.user/html/getting_started/main_workflow.html"
+		target="_blank">Absolute beginners should start here</a>
+
+	<p>The <i>Getting Started</i> section contains instructions on how to use the hale studio help and how to navigate the UI. It also contains tutorial sections on core workflows in hale studio.</p><br>
+	</br>
+
+	<a href="http://help.halestudio.org/latest/index.jsp"
+		target="_blank">Search for free text in the online help</a>
+
+	<p>The hale studio help page offers free text search to help you find the information you are looking for.</p><br>
+	</br>
+
+	<p>hale studio desktop software offers a number of example transformation projects that you can load directly. The example projects contain mappings which use a variety of the functions available in hale studio.
+	The example projects are a great way to gain familiarity with common use scenarios.The example projects can be accessed from the <i>Help</i> menu in hale studio.</p><br>
+	</br>
+
+	<div>
+		<img style="text-align: center; max-width: 95%" src="example_projects.png">
+	</div><br>
+	</br>
+
+
+	<a href="https://www.youtube.com/watch?v=BKNMV-Jp9HM"
+		target="_blank">Watch the introductory webinar</a>
+
+	<p>wetransform offers webinars on challenging topics throughout the year. Register for a webinar <a href="https://www.wetransform.to/services/workshops/"
+		target="_blank">on our website</a>.</p><br>
+	</br>
+
+	<a href="https://discuss.wetransform.to/#/forum"
+		target="_blank">Use the Support Forum</a>
+
+	<p>We offer a support forum where you can post questions and view discussions going on in the user community.</p><br>
+	</br>
+
+	<p>Use the chat function on our website. The chat icon is available in the bottom right corner on every page on our website. You will be instantly connected with a member of our service team.</p><br>
```

Comment:
Do we offer the chat functionality as support for hale studio users? I think the forum is the right place to ask such questions or otherwise, if there is an existing support account, you will use the service desk.

Reply:
Removed!